compiler/src/iree/compiler/Codegen/Common/GPU/Passes.td

```diff
@@ -84,11 +84,18 @@ def GPUConvertToCoalescedDMAPass :
   let summary = "Convert operations to coalesced DMA operations.";
   let dependentDialects = [
     "::mlir::iree_compiler::IREE::GPU::IREEGPUDialect",
-    "::mlir::arith::ArithDialect",
     "::mlir::gpu::GPUDialect",
     "::mlir::scf::SCFDialect",
-    "::mlir::tensor::TensorDialect",
-    "::mlir::vector::VectorDialect"
+    "::mlir::tensor::TensorDialect"
+  ];
+}
+
+def GPULowerCoalescedDMAToGlobalLoadsPass :
```

Comment:
This pass is misnamed - it's lowering to target DMA operations.

What you'll want is a AMD-specific pass targeting amdgpu::gather_to_lds` and then a pass with a name like this for the fallback

Reply:
renamed.